pinakes/main/catalog/models.py

```diff
@@ -553,7 +553,7 @@ class State(models.TextChoices):
         APPROVED = "approved"
         CANCELED = "canceled"
         DENIED = "denied"
-        FAILED = "failed"
+        FAILED = "error"
```

Comment:
All `FAILED` states in `Order`, `OrderItem`, `ApprovalRequest` and `Request` in approval model are all defined as `[Ff]ailed`. It's so weird to change to `error` here, better to modify in the `HandleApprovalEvent` side.